utils/utils.py

```diff
@@ -2,6 +2,11 @@
 import logging
 import math
 
+import requests
+
+BANDIT_APP_CREDS_PATH = "credentials/bandit_app_creds.json"
```

Comment:
i assume credentials/* is in .gitignore?

Reply:
yup